app/views/pipelines/_run_pipeline.html.erb

```diff
@@ -126,6 +126,25 @@
           </div>
         </div>
 
+         <div class='col-5'>
+          <%= form.label :run_enrichment_concurrently, class: 'form-label' do %>
+            Run Enrichment Concurrently
+
+            <span
+              data-bs-toggle="tooltip"
+              data-bs-title="Selecting this option when running your pipeline will run multiple enrichments at once"><i class="bi bi-question-circle" aria-label="helper text"></i>
```

Comment:
This tooltip isn't correct, it's more that the extractions within an enrichment will run in parallel. Even without this option multiple enrichments will run at once :)